demos/smart_classroom_demo_gapi/include/action_detector.hpp

```diff
@@ -54,28 +52,9 @@ using SSDHeads = std::vector<SSDHead>;
 /**
 * @brief Config for the Action Detection model
 */
-struct ActionDetectorConfig : public CnnConfig {
-    explicit ActionDetectorConfig(const std::string& path_to_model)
-        : CnnConfig(path_to_model) {}
-
-    /** @brief Name of output blob with location info */
-    std::string old_loc_blob_name{"mbox_loc1/out/conv/flat"};
-    /** @brief Name of output blob with detection confidence info */
-    std::string old_det_conf_blob_name{"mbox_main_conf/out/conv/flat/softmax/flat"};
-    /** @brief Prefix of name of output blob with action confidence info */
-    std::string old_action_conf_blob_name_prefix{"out/anchor"};
-    /** @brief Name of output blob with priorbox info */
-    std::string old_priorbox_blob_name{"mbox/priorbox"};
-
-    /** @brief Name of output blob with location info */
-    std::string new_loc_blob_name{"ActionNet/out_detection_loc"};
-    /** @brief Name of output blob with detection confidence info */
-    std::string new_det_conf_blob_name{"ActionNet/out_detection_conf"};
-    /** @brief Prefix of name of output blob with action confidence info */
-    std::string new_action_conf_blob_name_prefix{"ActionNet/action_heads/out_head_"};
-    /** @brief Suffix of name of output blob with action confidence info */
-    std::string new_action_conf_blob_name_suffix{"_anchor_"};
-
+struct ActionDetectorConfig {
+    /** @brief Person detection action recognition 0006 network enable flag */
+    bool new_network = false;
```

Comment:
Not sure if `new_network` is a good name in a new code. This `new_` / `old_` stuff could be there due to historical reasons

Reply:
New name: `net_with_six_actions`, because person-detection-action-recognition-0006 recognize six actions (0005 - 4).